createFolders.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(readlink "$0")")"
+
+if [[ $# -lt 1 ]]; then
+  echo "Usage: bash createFolders.sh <programming language>" >&2
```

Comment:
WDYT about renaming this file to `createDirectories` for consistency with the POSIX terminology?

Reply:
I think that is fair to have it be consistent with POSIX terminology